app/assets/javascripts/controllers/current_deploys_badge_ctrl.js

```diff
@@ -1,33 +1,30 @@
 samson.controller('currentDeployBadgeCtrl', function($scope, $http, SseFactory) {
   'use strict';
 
-  $scope.currentActiveDeploys = 0;
+  var $badge = $('#currentDeploysBadge');
+  $scope.currentActiveDeploys = parseInt($badge.data('count'), 10);
 
-  SseFactory.on('deploys', function(msg) {
-    if (msg.type === 'new') {
-      $scope.currentActiveDeploys += 1;
-    } else if (msg.type === 'finish') {
-      $scope.currentActiveDeploys -= 1;
-    }
-    updateBadge();
-  });
+  // connect to stream for updates once a user has the page open for a while
+  setTimeout(function(){
+    SseFactory.on('deploys', function(msg) {
+      if (msg.type === 'new') {
+        $scope.currentActiveDeploys += 1;
+      } else if (msg.type === 'finish') {
+        $scope.currentActiveDeploys -= 1;
+      }
+      updateBadge();
+    });
+  }, 5000);
 
   function updateBadge() {
     _.defer(function() { $scope.$apply(); });
     if ($scope.currentActiveDeploys > 0) {
-      $('#currentDeploysBadge').show();
+      $badge.show();
     } else {
       $scope.currentActiveDeploys = 0;
-      $('#currentDeploysBadge').hide();
+      $badge.hide();
     }
   }
 
-  function init() {
-    $http.get('/deploys/active_count.json').success(function(result) {
```

Comment:
I remember the live counter was explicitly requested by a few folks (I think mostly HC devs?).
So people are/were definitely using it. And I also check it every now and again myself.

We could change the 
- Put back in the ```$http.get('/deploys/active_count.json')``` line when the timer kicks in
- Or I remember the live counter was explicitly requested by a few folks (I think mostly HC devs?).
So people are/were definitely using it. And I also check it every now and again myself.

We could change the deploy SSE msgs to instead of sending a increment/decrement type msg, have it send the current total number of active deploys. Which would be helped by your cache entry you added for the deploy count.

Or else put back in the ```$http.get('/deploys/active_count.json')``` line when the timer kicks in, and at least have the benefit of reducing the load on the initial page load?